src/pages/MyNotes/components/Note/NoteContainer.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from 'react';
+
+import { INoteContainer } from 'pages/MyNotes/types';
+
+import Note from './Note';
+
+const NoteContainer: FC<INoteContainer> = ({
+  note,
+  activeNote,
+  setActiveNote,
+  editNote,
+}) => {
+  const isActiveNote = activeNote?.id === note?.id;
+
+  const handleActiveNote = () => {
+    if (isActiveNote) {
+      localStorage.setItem('activeNote', JSON.stringify(null));
+      setActiveNote(null);
+    } else {
+      localStorage.setItem('activeNote', JSON.stringify(note));
+      setActiveNote(note);
+    }
+  };
```

Comment:
add separate functions for each part of if/else statement and use ternary operator like isActiveNote ? func1 : func2